recipe/bld.bat

```diff
@@ -1,4 +1,11 @@
-move bin\* %LIBRARY_BIN%\
+REM Move only desired files as the downloaded package contains even Qt5 dlls
+move bin\cmake.exe %LIBRARY_BIN%\
+move bin\cmcldeps.exe %LIBRARY_BIN%\
+move bin\cpack.exe %LIBRARY_BIN%\
+move bin\ctest.exe %LIBRARY_BIN%\
+move bin\msvcp120.dll %LIBRARY_BIN%\
```

Comment:
My impulse would be "no".  Those are system dlls, right?  Or maybe they need to be in a separate package.  Maybe try this PR without it and see if the tests pass?